src/Mod/Sketcher/Gui/TaskSketcherElements.h

```diff
@@ -142,9 +154,35 @@ public Q_SLOTS:
     int previouslySelectedItemIndex;
 
     bool isNamingBoxChecked;
-    bool isautoSwitchBoxChecked;
 
-    bool inhibitSelectionUpdate;
+    const MultIcon Sketcher_Element_Arc_Edge = MultIcon("Sketcher_Element_Arc_Edge");

```

Comment:
These should not be in the class definition. They are also created every time a TaskSketcherElements object is created. As an implementation detail, I think they should be made into a separate class in the cpp file. Maybe into a singleton or a factory. Will come back with a proposal.

Reply:
My bad. Originally, it was just declared here but thus used an empty constructor which was not ideal.
I agree that MultIcon should move to separate class that handles caching through a singleton. I noted that for myself but had no time to work on this. :/